packages/debug/src/browser/debug-frontend-application-contribution.ts

```diff
@@ -1197,7 +1197,7 @@ export class DebugFrontendApplicationContribution extends AbstractViewContributi
     }
 
     async start(noDebug?: boolean, debugSessionOptions?: DebugSessionOptions): Promise<void> {
-        let current = debugSessionOptions ? debugSessionOptions : this.configurations.current;
+        let current = debugSessionOptions && !isAnchor(debugSessionOptions) ? debugSessionOptions : this.configurations.current;
```

Comment:
It would be better to check this condition in the command handler (around line 640), since the possibility of receiving of anchor is limited to that context.

Reply:
> It would be better to check this condition in the command handler (around line 640), since the possibility of receiving of anchor is limited to that context.

ok， done